### PR TITLE
ContestSummaryMenu improvements 

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* ContestSummaryMenu improvements

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt [hunk 1]
```diff
@@ -43,7 +43,7 @@ private fun ChildrenBuilder.displayTopProjects(sortedResults: List<ContestResult
                 "${index + 1}. ",
                 contestResult.projectName,
                 contestResult.organizationName,
-                contestResult.score?.toString() ?: "-",
+                (contestResult.score?.toString().asDynamic().toFixed(2) as String? ?: "-"),
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: Why `asDynamic()`?

**Review by sanyavertolet — COMMENTED** [review]
Please provide a screenshot

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: Kotlin native have no methods for decimal precisions, but kotlin JS does: https://stackoverflow.com/questions/42791492/kotlin-js-define-number-of-decimals

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: ok, weird that Kotlin does not have it

Comment (kgevorkyan) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: Only kotlin jvm does, other seems to be absent and issue is still opened for more than 5 years https://youtrack.jetbrains.com/issue/KT-21644

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt [hunk 2]
```diff
@@ -43,7 +43,7 @@ private fun ChildrenBuilder.displayTopProjects(sortedResults: List<ContestResult
                 "${index + 1}. ",
                 contestResult.projectName,
                 contestResult.organizationName,
-                contestResult.score?.toString() ?: "-",
+                (contestResult.score?.asDynamic().toFixed(2) as String?) ?: "-",
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: We already have an extension for it: https://github.com/saveourtool/save-cloud/blob/11117025df8326159aeb35c61f176dab51b11971/save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/Utils.kt#L147

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt:46]: https://github.com/saveourtool/save-cloud/pull/1431

**Pull request merged**